app/controllers/videos_controller.rb

```diff
@@ -23,6 +24,8 @@ def create
       flash[:error] = @video.errors.full_messages.join(", ")
       redirect_to :back
     end
+
+    VideoMailer.upload(@video.file, 'user@user.com')
```

Comment:
Good question. I was planning on using the filename and forgot to remove. I'll update.